### DISCUSSION TITLE
Display of time in multiple time zones

### DISCUSSION THREAD

**Discussion by seena417** [martinet101/ElevenClock#1152] [Q&A]
Thanks for the nice application! I use it conveniently.
I just have one request.
I would like to display the local time and the time in another time zone at the same time.
For example, the first line should be Japan Standard Time and the second line should be UTC time.
Please consider this.

**Comment by marticliment** [comment]
You can add a second clock on the other side of your taskbar, customize it individually and use the internet time feature with a custom url to get your desired timezone

**Comment by arshanskiyav** [comment]
> Thanks for the nice application! I use it conveniently. I just have one request. I would like to display the local time and the time in another time zone at the same time. For example, the first line should be Japan Standard Time and the second line should be UTC time. Please consider this.

Hello, I have the same problem, I found a solution for myself and wrote about it [here](https://github.com/marticliment/ElevenClock/discussions/1255)
